server/schemas/field.graphql

```diff
@@ -227,10 +240,12 @@ input SchemaFieldTypePropertyInput @onlyOne {
   integer: SchemaFieldIntegerInput
   reference: SchemaFieldReferenceInput
   url: SchemaFieldURLInput
+  group: SchemaFieldGroupInput
 }
 
 input CreateFieldInput {
-  modelId: ID!
+  modelId: ID
+  groupId: ID
```

Comment:
```suggestion
  schemaId: ID
```

Reply:
I totally agree, but this change requires a lot of changes in our current implementation. Unfortunately, we have a critical deadline this time
I'm planning to refactor this part later